Menus/MainMenu/MainMenu.cs

```diff
@@ -121,6 +109,25 @@ void Start()
 			UpdateToolButtons();
 		}
 
+		public void SetVisible(bool visible, bool temporary = false)
+		{
+			if (m_Visible != visible)
+			{
+				m_Visible = visible;
+				if (m_MainMenuUI)
+				{
+					m_MainMenuUI.visible = visible;
+					if (!temporary)
+						SendVisibilityPulse();
+				}
+			}
+		}
+
+		public bool GetVisible()
```

Comment:
with the flags-enum route you could go back to a property and in fact a singular one